content/tracing/faq.md

```diff
@@ -17,7 +17,7 @@ customnav: tracingnav
 
 ### How long is tracing data stored?
 
-Tracing data is stored for about 36 hours.
+Traces will be probabilistically available for up to 6 months, unless viewed. If you view a Trace, it is guaranteed to be stored for 6 months. 
```

Comment:
I think "probabilistically available" sounds a little confusing.  Maybe "Traces are stored for up to 6 months; the storage time for each trace is determined probabilistically.  If you view a trace, it is guaranteed to be stored for 6 months."